boards/arm/arduino_zero/Kconfig.defconfig

```diff
@@ -6,3 +6,7 @@
 config BOARD
 	default "arduino_zero"
 	depends on BOARD_ARDUINO_ZERO
+
+config SPI_4
+	default y
```

Comment:
Remove `SPI_4` here, this should have no impact as the only driver `SPI_4` has meaning on is the nrfx SPI driver.

Reply:
Done